src/node_internals.h

```diff
@@ -356,6 +356,8 @@ int ThreadPoolWork::CancelWork() {
   return uv_cancel(reinterpret_cast<uv_req_t*>(&work_req_));
 }
 
+void StartTracingAgent();
+
```

Comment:
teeny tiny nit: I’d remove this line, to keep the “platform methods” grouped together